case/src/main/kotlin/com/ritense/case/service/CaseDefinitionService.kt

```diff
@@ -386,7 +403,18 @@ class CaseDefinitionService(
         }
     }
 
+    private fun validatePath(path: String, key: String) {
```

Comment:
I would rename this to validateExportPath because i believe this is only relevant in this situation.